app/src/services/deploymentHistory.js

```diff
@@ -60,5 +60,30 @@ module.exports = {
       return this.getLatestEnvAction(acronymId, env);
     }));
     return results.filter(x => !!x);
+  },
+
+  /**
+   * @function findHistory
+   * Returns an array of the deployment history for an acronym, joined with the user records
+   * @param {string} acronym The desired acronym
+   * @returns {object[]} An array of history object
+   */
+  async findHistory(acronym) {
+    return await db.Acronym.findOne({
```

Comment:
Do we need the async/await wrapper here? Looks like a singleton operation so could just return the promise of the findOne operation instead (can't remember the general pattern we did in getok).

Reply:
All the other services look like they await the query and return that result so probably would stick with that for this one too.